src/sotoki/utils/imager.py

```diff
@@ -153,7 +153,8 @@ def get_s3_key_for(self, url: str) -> str:
 
     def get_digest_for(self, url: str) -> str:
         """Unique identifier of that url"""
-        return zlib.adler32(url.encode("UTF-8"))
+        return hashlib.md5(url.encode("UTF-8"), usedforsecurity=False).hexdigest()
```

Comment:
`usedforsecurity` is 3.9+ only while sotoki currently uses `3.8` in the Image and even (probably incorrectly) targets `3.6+` in the setup. Can you remove the param?